corehq/apps/export/models/new.py

```diff
@@ -2675,6 +2675,12 @@ def delete(self, using=None, keep_parents=False):
         return super(DataFile, self).delete(using, keep_parents)
 
 
+class EmailRequest(models.Model):
+    domain = models.CharField(max_length=255)
+    download_id = models.CharField(max_length=255)
+    email_address = models.CharField(max_length=255)
```

Comment:
While we may only need an email address here, I think it would be better to put the user's `user_id` and then use that to look up the user and finally get the email address once you're in the task. There are a couple reasons for this. First, the user is the richer context from which you can derive the `email_address`—if later on we need to know other things about the user (like their language preference for instance) it will be nice to have available.

If you don't change this now, that's not the end of the world, but if it's not too hard, I think it'd be a minor improvement.